mozfldp/server.py

```diff
@@ -4,16 +4,21 @@
 
 from flask import Flask
 from flask import request
+import numpy as np
 import argparse
 
 # TODO: this needs to manually copied in right now.
 
 # we should package up the canosp project as a pypi project so we can
 # just import it like any other module.
 
-from mozfldp.simulation_util import server_update  # noqa
 
-from sklearn.linear_model import SGDClassifier
+
+def append(list, element):
+    """
+    helper function to append array into array in numpy
+    """
+    return np.concatenate((list, [element])) if list is not None else [element]
```

Comment:
This is a bit awkward. Maybe just initialize enough storage beforehand?

Reply:
Done! Switched to regular python arrays instead of using numpy 😄